opensearch/Dockerfile

```diff
@@ -1,4 +1,4 @@
 FROM opensearchproject/opensearch:2.11.1
 
 # Install the opensearch-ubl plugin.
-RUN /usr/share/opensearch/bin/opensearch-plugin install https://github.com/o19s/opensearch-ubl/releases/download/0.0.5/opensearch-ubl.zip
+RUN /usr/share/opensearch/bin/opensearch-plugin install https://github.com/o19s/opensearch-ubl/releases/download/0.0.6/opensearch-ubl.zip
```

Comment:
can we update to opensearch-ubi here?   Long live UBI.

Reply:
Yes, good catch.